CIP-0139/README.md

```diff
@@ -0,0 +1,77 @@
+---
+CIP: 139
```

Comment:
```suggestion
CIP: ?
```

Reply:
thanks for changing your "rendered document" link as well... usually editors have to do that 🙏